app/controllers/public_health_controller.rb

```diff
@@ -74,88 +51,6 @@ def tab_counts
     render json: { total: patients.size }
   end
 
-  protected
```

Comment:
I removed this `protected` after moving this methods - any reason it was here at all @timwongj?

Reply:
I think `protected` might have been added by someone else before and it probably just showed up as a diff I created when working on the linelists, I don't have a particular reason for it being here

Reply:
Ah okay, thanks!

Reply:
Put any methods that aren't hit directly by a route below `protected` (not a bad idea in general for controllers)